modules/imgproc/perf/opencl/perf_imgwarp.cpp

```diff
@@ -72,7 +72,7 @@ OCL_PERF_TEST_P(WarpAffineFixture, WarpAffine,
     const WarpAffineParams params = GetParam();
     const Size srcSize = get<0>(params);
     const int type = get<1>(params), interpolation = get<2>(params);
-    const double eps = CV_MAT_DEPTH(type) <= CV_32S ? 1 : interpolation == INTER_CUBIC ? 2e-3 : 1e-4;
+    const double eps = CV_MAT_DEPTH(type) <= CV_32S ? 2 : interpolation == INTER_CUBIC ? 2e-3 : 3e-2;
```

Comment:
It is tuned because CI hosts, which are connected via GHA, can build opencv with opencl, but somehow opencl is not available in the runtime. Guess we need to pass GPU device when running docker.

cc @asmorkalov

Reply:
@asmorkalov, the new algorithm and the old algorithm are still different, even with `algorithmHint==ALGO_HINT_ACCURATE`. So I don't see anything criminal with raising eps from 1 to 2